### PR TITLE
Javadoc ndg

### PR DESCRIPTION
**Pull request by wwweert123**

### CODE REVIEW

File: src/test/testData/incorrectwatchlist.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "exchange": "NASDAQ",
+    "stockName": "Apple Inc.",
+    "price": "186.4",
+    "dayHigh": "186.565",
+    "dayLow": "183.53",
+    "lastUpdated": "Nov 11, 2023, 5:00:01 AM",
+    "lastFetched": 1699698248423,
+    "hashCode": 0
+  }
+}
```
Comment (NeoMinWei) [inline src/test/testData/incorrectwatchlist.json:24]: Good extensive testing

File: src/main/java/seedu/financialplanner/investments/WatchList.java [hunk 2]
```diff
@@ -255,7 +263,7 @@ public String addStock(String stockCode) throws FinancialPlannerException {
         }
 
         Stock newStock;
-        newStock = new Stock(stockCode);
+        newStock = new Stock(stockCode.toUpperCase());
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/investments/WatchList.java:266]: good fix

**Review by NeoMinWei — APPROVED** [review]
LGTM

**Review by ryan1604 — APPROVED** [review]
LGTM

**Pull request merged**